readthedocs/templates/core/project_details.html

```diff
@@ -41,6 +42,16 @@ <h3>{% trans "Versions" %}</h3>
           {% if request.user|is_admin:project %}
             <ul class="module-item-menu">
               <li><a href="{% url "project_version_detail" project.slug version.slug %}">{% trans "Edit" %}</a></li>
+              <li>
+                <span class="right quiet status-middle">
+                  {% is_latest_built_success version as is_success %}
```

Comment:
This should use this model method: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/models.py#L882 -- it's likely much more efficient.

Reply:
`get_latest_build` returns the latest built of the project -- irrespective of the version.
We need the builds to be of specific version here.